Sources/FigmaAPI/Model/Node.swift

```diff
@@ -38,10 +38,37 @@ public struct Document: Decodable {
     public let style: TypeStyle?
 }
 
+// https://www.figma.com/plugin-docs/api/Paint/
 public struct Paint: Decodable {
-    public let type: String
+    public let type: PaintType
+    public let opacity: Double?
+    public let color: PaintColor?
+
+    public var asSolid: SolidPaint? {
+        return SolidPaint(self)
+    }
+}
+
+public enum PaintType: String, Decodable {
+    case solid = "SOLID"
+    case image = "IMAGE"
+    case rectangle = "RECTANGLE"
+    case gradientLinear = "GRADIENT_LINEAR"
+    case gradientRadial = "GRADIENT_RADIAL"
+    case gradientAngular = "GRADIENT_ANGULAR"
+    case gradientDiamond = "GRADIENT_DIAMOND"
```

Comment:
Why did you add gradient types of colors? Android and iOS doesn't support gradient colors.

Reply:
In order to prevent decoding errors from decoding `PaintType`. Those are all possible values of `PaintType` according to the Figma document. Say there's no "GRADIENT_LINEAR" defined in the `PaintType` whereas one figma project includes it. Then the `PaintType` will fail to decode "GRADIENT_LINEAR" as the enum doesn't expect it. That's why I put all those values even though we're only using `SOLID` at this point.

Alternatively, we can get rid of `PaintType` and leave `type` as `String` type as it used to be. Then check if the value is "SOLID" before we access to the `color`. I think this could be shorter and simpler way.